_pages/en_US/get-started.txt

```diff
@@ -76,13 +76,13 @@ There have been reports of a wave of bans being handed out to CFW users by Ninte
       <td style="text-align: center; font-weight: bold;"><a href="homebrew-launcher-(soundhax)">Homebrew Launcher (Soundhax)</a></td>
```

Comment:
Doesn't the sidebar navigation/method tracking also need to be updated? Not that that many people look at it, at least I think..

Reply:
probably, yes, but i thought that was dynamically managed by the page/nav layout? where do i need to make changes?

Reply:
Honestly not sure, I just know that PicHaxx still links to Fredtool in the sidebar for whatever reason

Reply:
i fixed this!